cmd/libs/go2idl/types/types.go

```diff
@@ -275,6 +275,18 @@ var (
 		Name: Name{Name: "uint"},
 		Kind: Builtin,
 	}
+	Float64 = &Type{
```

Comment:
Wow, I totally forgot about floats! We can keep pretending complex types don't exist ;)